src/SignalR/clients/ts/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj

```diff
@@ -17,6 +17,13 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.targets))\Directory.Build.targets" />
 
+  <Target Name="Build" DependsOnTargets="$(BuildDependsOn)">
+    <Yarn Command="install --mutex network"
```

Comment:
This shouldn't be needed right? Install should happen in restore

Reply:
Hmm maybe not after I added the BuildDependsOn. Made the fix in separate commits so I forgot to remove the wrong one.